### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by Godedict**
Portfolio: finish mobile version project was implemented using the following requirement:

- Make sure the content of the "About myself" section is aligned with flexbox.

- In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

- Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript should be list items.

- Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

- You don't need to implement the detail modal window that requires JavaScript.

- You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Implement the button interactions (enable, hover, etc.).

- The website should be responsive, following these breakpoints:

- Minimum mobile screen size should be 375px

- Minimum desktop screen size should be 768px

- Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +42,155 @@ <h1>I'm Aderounmu.<br>Glad to see<br>you!</h1>
       </ul>
     </div>
   </section>
+
+  <section class="page-2">
```
Comment (souad988) [inline index.html]: great work done so far 👍🏼  , but the section tag is meant to wrap all project cards as all these items are same 
a section is a semantic element for creating standalone sections in a web page. These sections should be made up of related content, like contact information. The section element should only be used if there isn't a more specific element to represent the related content.


Also this section needs to have a display grid to position it's items, and the background color is diferent than the one udes in figma ==> background: #F7F7F9;

File: index.html [hunk 2]
```diff
@@ -42,6 +42,155 @@ <h1>I'm Aderounmu.<br>Glad to see<br>you!</h1>
...
+        </ul>
+        <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+      </div>
+      <ul class="btn">
+        <li><a href="#" class="html">HTML</a></li>
+        <li><a href="#" class="css">CSS</a></li>
+        <li><a href="#" class="javascript">JavaScript</a></li>
+      </ul>
+      <div class="container">
+        <a href="#">See project</a>
```
Comment (souad988) [inline index.html]: Kindly add hover an press effects to all buttons as shown in figma design

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Godedict  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Design well implemented
Good file structure

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- condensing pr sammary content as it should be short but includes main details 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Godedict) [reply index.html]: Done, thank you.

File: index.html (hunk 2 above)
Comment (Godedict) [reply index.html]: Change implemented, thank you.

File: index.html [hunk 3]
```diff
@@ -42,6 +42,157 @@ <h1>I'm Aderounmu.<br>Glad to see<br>you!</h1>
       </ul>
     </div>
   </section>
+
+  <section class="card">
```
Comment (bienvenuushindi) [inline index.html:46]: :red_circle: Please make sure you apply the grid layout on this section wrapper as [requested by the previous reviewer](https://github.com/Godedict/Portfolio-Setup/pull/3#discussion_r1214569935). This means the CSS display property of this element should be equal to  grid.

File: style.css [hunk 4]
```diff
@@ -7,7 +7,7 @@
 body {
   font-family: 'Poppins', sans-serif;
   font-weight: 900;
-  background-color: #e5e5e5;
+  background-color: #f7f7f9;
 }
 
 .buttons {
```
Comment (bienvenuushindi) [inline style.css:13]: :red_circle: As [requested by the previous reviewer](https://github.com/Godedict/Portfolio-Setup/pull/3#discussion_r1214570499), Please make sure your button hover state design is the same as the one on Figma. As you can see below there is a huge gap between  the hover state on your Design and the Figma

Your Design
your

Figma (The second button)
buttons

Note: Kindly select the right background color from the Figma Template

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @Godedict      ,

Good job implementing most of the requested  changes!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (Godedict) [reply index.html:46]: Changes made

File: style.css (hunk 4 above)
Comment (Godedict) [reply style.css:13]: changes made.. thanks

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @Godedict :wave: 
 
Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: Descriptive pull request.
:heavy_check_mark: Correct GitHub flow.
:heavy_check_mark: Commit messages are meaningful.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**